quantum/action_tapping.c

```diff
@@ -164,15 +164,14 @@ bool process_tapping(keyrecord_t *keyp) {
                  * useful for long TAPPING_TERM but may prevent fast typing.
                  */
                 // clang-format off
-#    if defined(TAPPING_TERM_PER_KEY) || (TAPPING_TERM >= 500) || defined(PERMISSIVE_HOLD) || defined(PERMISSIVE_HOLD_PER_KEY) || (defined(AUTO_SHIFT_ENABLE) && defined(RETRO_SHIFT))
+#    if defined(PERMISSIVE_HOLD) || defined(PERMISSIVE_HOLD_PER_KEY) || (defined(AUTO_SHIFT_ENABLE) && defined(RETRO_SHIFT))
                 else if (
                     (
                         (
-                            GET_TAPPING_TERM(tapping_keycode, &tapping_key) >= 500
 #        ifdef PERMISSIVE_HOLD_PER_KEY
-                            || get_permissive_hold(tapping_keycode, &tapping_key)
+                            get_permissive_hold(tapping_keycode, &tapping_key)
 #        elif defined(PERMISSIVE_HOLD)
-                            || true
+                            true
 #        endif
```

Comment:
That was a bit rushed, the unit-tests are failing now with:

```
Making test caps_word_autoshift                                                                                [ERRORS]
quantum/action_tapping.c: In function ‘process_tapping’:
quantum/action_tapping.c:176:25: error: expected expression before ‘)’ token
  176 |                         ) && IS_RELEASED(event) && waiting_buffer_typed(event)
      |                         ^
 | 
 | 
 | 
```

I guess we need a false at this position for all other cases?

```suggestion
                            true
#        else
                            false                     
#        endif
```

Reply:
Thanks!